cli/internal/run/run.go

```diff
@@ -893,12 +845,24 @@ func (e *execContext) exec(pt *nodes.PackageTask, deps dag.Set) error {
 	return nil
 }
 
-func (r *run) generateDotGraph(taskGraph *dag.AcyclicGraph, outputFilename fs.AbsolutePath) error {
-	graphString := string(taskGraph.Dot(&dag.DotOpts{
-		Verbose:    true,
-		DrawCycles: true,
-	}))
+func (r *run) generateDotGraph(graphString string, graphVizWarn bool) {
```

Comment:
This, and the `HasGraphViz` utility could maybe stand to be pulled into their own package.

Reply:
Yea agree, I would like to move more things here